cmd/pebble/main.go

```diff
@@ -71,11 +72,26 @@ func main() {
 	wfeImpl := wfe.New(logger, db, va, ca, *strictMode)
 	muxHandler := wfeImpl.Handler()
 
+	if c.Pebble.ManagementListenAddress == "" {
+		go func() {
+			adminHandler := wfeImpl.ManagementHandler()
+			err = http.ListenAndServeTLS(
+				c.Pebble.ManagementListenAddress,
+				c.Pebble.Certificate,
+				c.Pebble.PrivateKey,
+				adminHandler)
+			cmd.FailOnError(err, "Calling ListenAndServeTLS() for admin interface")
+		}()
+		logger.Printf("Management interface listening on: %s\n", c.Pebble.ManagementListenAddress)
+		logger.Printf("Root CA certificate(s) available at: https://%s%s",
```

Comment:
I would either write
```suggestion
		logger.Printf("Root CA certificate available at: https://%s%s0",
```
or something like
```suggestion
		logger.Printf("Root CA certificate(s) available at: https://%s%s<no>, where <no> is 0, 1, ...",
```
(or even include `alternateRoots` in here)

Reply:
I implemented your last proposition: if several alternate roots are generated, each of them has a dedicated log to display its dedicated `root` endpoint.

Reply:
Thanks for the :mag: @felixfontein !